pkg/crc/constants/constants.go

```diff
@@ -22,18 +22,19 @@ const (
 
 	CrcEnvPrefix = "CRC"
 
-	ConfigFile                = "crc.json"
-	LogFile                   = "crc.log"
-	DaemonLogFile             = "crcd.log"
-	CrcLandingPageURL         = "https://console.redhat.com/openshift/create/local" // #nosec G101
-	DefaultAdminHelperURLBase = "https://github.com/crc-org/admin-helper/releases/download/v%s/%s"
-	DefaultBundleURLBase      = "https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/%s/%s/%s"
-	DefaultContext            = "admin"
-	DaemonHTTPEndpoint        = "http://unix/api"
-	DaemonVsockPort           = 1024
-	DefaultPodmanNamedPipe    = `\\.\pipe\crc-podman`
-	RootlessPodmanSocket      = "/run/user/1000/podman/podman.sock"
-	RootfulPodmanSocket       = "/run/podman/podman.sock"
+	ConfigFile                    = "crc.json"
+	LogFile                       = "crc.log"
+	DaemonLogFile                 = "crcd.log"
+	CrcLandingPageURL             = "https://console.redhat.com/openshift/create/local" // #nosec G101
+	DefaultAdminHelperURLBase     = "https://github.com/crc-org/admin-helper/releases/download/v%s/%s"
+	CRCWin32BackgroundLauncherURL = "https://github.com/crc-org/win32-background-launcher/releases/download/v%s/%s"
```

Comment:
I don't think the `CRC` prefix is really useful here, and I'd also drop "win32". This does not need to be exported.